packages/superset-ui-core/test/query/getColumnLabel.test.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { getColumnLabel } from '@superset-ui/core/src/query';
```

Comment:
Shouldn't we import from `lib` rather than `src`? Also, can we export it from `core` soon so that we can shorten that import?

Reply:
oops, that's a mistake

Reply:
Ah, I see that it's already exported from `@superset-ui/core`. Can we shorten that import?

Reply:
hmm, for some reason `importing from `superset-ui/core` caused codecov to miss test coverage. I'll keep it unchanged for now and defer this to a later fix.